pytorch_lightning/core/lightning.py

```diff
@@ -1571,12 +1572,17 @@ def _auto_collect_arguments(cls, frame=None) -> Tuple[Dict, Dict]:
             parents_arguments.update(args)
         return self_arguments, parents_arguments
 
-    def save_hyperparameters(self, *args, frame=None) -> None:
+    def save_hyperparameters(
+        self, *args, ignore: Optional[Union[List[str], str]] = None, frame: Optional[types.FrameType] = None
+    ) -> None:
         """Save all model arguments.
 
         Args:
             args: single object of `dict`, `NameSpace` or `OmegaConf`
-             or string names or arguments from class `__init__`
+                or string names or arguments from class `__init__`
+            ignore: an argument or a list of arguments from class `__init__`
+                to be ignored
+            frame: a frame object. Default is None
```

Comment:
normally we put examples into an ``Example`` block (everything needs to be indendent then)
```suggestion
            frame: a frame object. Default is None
        
        Example::
```